views/components/GraphTable/GraphTable.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+import { formatSiteData, graphTableColumns } from '../../fe-utils/helpers'
+import { graphTableRowDataBySiteUtil } from './helpers'
+
+const GraphTable = ({ graph }) => {
+  if (!graph) return null
+
+  const tableColumns = graphTableColumns(graph.labels)
+  const [totalsSite, ...sites] = graphTableRowDataBySiteUtil(graph.dataBySite)
+
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {tableColumns.map(({ name }) => (
+              <React.Fragment key={name + 'table-header'}>
+                <TableCell align="center">{name}</TableCell>
+              </React.Fragment>
```

Comment:
We don't need to use a fragment here.

```suggestion
              <TableCell align="center" key={name}>{name}</TableCell>
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/193/commits/98ef1ad36740b7dcf6eae8a2e3f687620ac21cc2